src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styleObject = {};
+
+  const declarations = sourceString
+    .split(';')
+    .map((declaration) => declaration.trim());
+
+  declarations.forEach((declaration) => {
+    const [key, value] = declaration.split(':').map((part) => part.trim());
```

Comment:
[CODE STYLE] - All chained methods called on the same level should have the same indentation. This makes code much more readable. While you have not chained methods here, the concept applies to the array destructuring as well. Ensure that the destructuring and mapping are clearly readable.